parastell/invessel_build.py

```diff
@@ -226,39 +217,71 @@ def calculate_loci(self):
 
         [surface.calculate_loci() for surface in self.Surfaces.values()]
 
+    def _check_edge(self, edge):
+        """Checks a CadQuery edge to determine if the its center of mass is
+        located at either end of the stellarator build.
+        (Internal function not intended to be called externally)
+
+        Arguments:
+            edge (object): CadQuery.Edge object.
+
+        Returns:
+            (bool): flag to indicate whether edge is located at either end of
+                stellarator build.
+        """
+        x = edge.Center().x
+        y = edge.Center().y
+
+        toroidal_angle = np.arctan2(y, x)
+        """if self.radial_build.toroidal_angles[0] >= 0.0:
+            toroidal_angle = (toroidal_angle + 2 * np.pi) % (2 * np.pi)"""
```

Comment:
Do we need this?  Seems like we might want it, in general?

Reply:
I think I meant to delete that commented code block. In my local version of this branch (yet to be pushed) I created a utility function to check whether an input value is within a given range, accounting for a periodic domain, removing the need to ensure that angles are positive.